lib/stitches/auto_fill.py

```diff
@@ -47,18 +49,42 @@ def is_segment(self):
 
 
 @debug.time
-def auto_fill(shape,
-              angle,
-              row_spacing,
-              end_row_spacing,
-              max_stitch_length,
-              running_stitch_length,
-              running_stitch_tolerance,
-              staggers,
-              skip_last,
-              starting_point,
-              ending_point=None,
-              underpath=True):
+def auto_fill(shape: Shape,
+              angle: float,
+              row_spacing: int,
+              end_row_spacing: int,
+              max_stitch_length: float,
+              running_stitch_length: float,
+              running_stitch_tolerance: float,
+              staggers: int,
```

Comment:
all 3 int parameters should be float

Reply:
Why would staggers be a float? The code in `adjust_stagger` and `stitch_row` (https://github.com/inkstitch/inkstitch/blob/ea13c429973e50ef55ee4bf4403a0618cb9b26d8/lib/stitches/fill.py#L39-L47) seems to suggest it should be integral

>  # Each successive row of stitches will be staggered, with
>  # num_staggers rows before the pattern repeats.

Reply:
#1874

Reply:
Updated